tests/e2e/specs/modules/analytics/setup-gcp-no-account-no-tag.test.js

```diff
@@ -184,14 +192,19 @@ describe( 'setting up the Analytics module using GCP auth with no existing accou
 			'.googlesitekit-analytics__select-account .mdc-select__selected-text',
 			{ text: '' }
 		);
-		await expect( page ).toMatchElement(
-			'.googlesitekit-analytics__select-property .mdc-select__selected-text',
-			{ text: '' }
-		);
-		await expect( page ).toMatchElement(
-			'.googlesitekit-analytics__select-profile .mdc-select__selected-text',
-			{ text: '' }
-		);
+
+		await expect( page ).toClick( '.mdc-select', {
+			text: '',
+		} );
```

Comment:
Is there a better class we can use for this? The previous was specific to a particular input.